configure.meson

```diff
@@ -1,12 +1,11 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Why?

Reply:
because echo `-e` is used

Reply:
```
$ which echo
/usr/bin/echo
$ /usr/bin/echo -e "\taa"
	aa
```

So are you sure?

Reply:
The shell (sh) built-in, which gets used in preference to any external command, doesn't support it. The reason it worked for you is that there was no `#!` at the top of the script, so the script was run as a child of your current shell (presumably bash or some other enhanced shell). e.g.
~~~
ajb@calypso:/clients/common/fdpp.git$ sh
$ echo -e "\tbob"
-e 	bob
$ 
~~~

Reply:
Dropped the formatting and bash requirement.